develop/components/pages/TablePage.tsx

```diff
@@ -0,0 +1,228 @@
+
+                    <p>Here are sample outputs of plain table</p>
+                    <div className="result wide">
+                        <Table
+                            columns={columns}
+                            data={smallData}
+                        />
+                    </div>
+
+                    <p>Here an example with sorting</p>
+                    <div className="result wide">
+                        <Table
+                            columns={columns}
+                            data={smallData}
+                            sortProps={{
+                                onAfterSorting: (rows: Array<TableRow>, sortByColumn: TableHeader) => { console.log("The sorted rows are ", rows); }
+                            }
+                            }
+                        />
+                    </div>
+
+                    <p>Here an example with pagination</p>
+                    <div className="result wide">
+                        <Table
+                            columns={columns}
+                            data={data}
+                            offset={pageSize}
+                            currentpage={paginationValue}
+                            footer={
+                                <Pagination
+                                    value={paginationValue}
+                                    onChange={setPagination}
+                                    size={listSize}
+                                    useFirstAndLast={true}
+                                />
+                            }
+                        />
+                    </div>
+
+                    <p>Here is an example with expandable subrows and rowDetails</p>
+                    <div className="result wide">
+                        <Table
+                            columns={columns}
+                            data={smallData}
+                            onRowExpanded={(rows: Array<TableRow>) => { console.log("the expanded ros are ", rows); }}
+                        />
+                    </div>
+
+                    <p>Here is an example with row selection</p>
+                    <div className="result wide">
+                        <Table
+                            columns={columns}
+                            data={smallData}
+                            onRowSelected={(rows: Array<TableRow>) => { console.log("The selected rows are ", rows); }}
+                        />
+                    </div>
+
+                    <p>Here is an example with row selection and subRows</p>
+                    <div className="result wide">
+                        <Table
+                            columns={columns}
+                            data={smallData}
+                            onRowSelected={(rows: Array<TableRow>) => { console.log("The selected rows are ", rows); }}
+                            onRowExpanded={(rows: Array<TableRow>) => { console.log("the expanded ros are ", rows); }}
```

Comment:
I noticed you have a lot of console logs in this file. Console logs serve no purpose for the majority of the users. You shouldn't expect them to open the console to verify that certain aspects of the component are triggering the right methods. Scan this file and remove them all.